symengine/ntheory.cpp

```diff
@@ -1194,6 +1194,23 @@ bool _nthroot_mod1(std::vector<RCP<const Integer>> &roots, const integer_class &
     return true;
 }
 
+// Checks if Solution for x**n == a mod p**k exists where a != 0 mod p and p is an odd prime.
+bool is_nthroot_mod1(const integer_class &a, const integer_class &n,
```

Comment:
The functions not declared in the header files have names starting from `_`. Is that a symengine convention ?